vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/AbstractNumberField.java

```diff
@@ -47,6 +47,7 @@
  *
  * @author Vaadin Ltd.
  */
+@SuppressWarnings("deprecation")
```

Comment:
Let's add this to `BigDecimalField` as well.

Reply:
Addressed in the latest commit.